tests/servers/test_navigation_server_3d.h

```diff
@@ -123,6 +123,7 @@ TEST_SUITE("[Navigation]") {
 		CHECK_FALSE(map.is_valid());
 
 		SUBCASE("Queries against invalid map should return empty or invalid values") {
+			ERR_PRINT_OFF
```

Comment:
It's not technically needed, but style wise we prefer adding semicolons after such macros.